lib/Yam/Agama/Pom/GrubMenuSLESPage.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles grub screen in SLES.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Agama::Pom::GrubMenuSLESPage;
+use strict;
+use warnings;
+use testapi;
+
+sub new {
+    my ($class, $args) = @_;
+    return bless {
+        grub_menu_base => $args->{grub_menu_base},
+        tag_first_entry_highlighted => 'grub2',
```

Comment:
this is not supposed to be generic.

Reply:
It's the same tag (`grub2`) as found in test case [agama_default@64bit](https://openqa.suse.de/tests/15682988#step/grub_test/1)

Reply:
we are trying to be specific with this new code, so we can fix expectation about the product we want to find.